UI/UnoZIndex/UnoZIndex/UnoZIndex/UnoZIndex/Presentation/MainPage.xaml.cs

```diff
@@ -1,23 +1,29 @@
-﻿using Uno.UI.Toolkit;
+﻿using Uno.Toolkit.UI;
 
 namespace UnoZIndex.Presentation
 {
     public sealed partial class MainPage : Page
     {
+        ShadowContainer foregroundCard;
+
         public MainPage()
         {
             this.InitializeComponent();
+            foregroundCard = Card4;
         }
 
-        private void ElevatedView_Tapped(object sender, Microsoft.UI.Xaml.Input.TappedRoutedEventArgs e)
+        private void Card_Tapped(object sender, Microsoft.UI.Xaml.Input.TappedRoutedEventArgs e)
         {
-            var card = sender as ElevatedView;
-            if(card is not null)
+
+            var card = sender as ShadowContainer;
+            Canvas.SetZIndex(foregroundCard, 0);
+            if (card is not null)
             {
                 var zIndex = Canvas.GetZIndex(card);
-                Canvas.SetZIndex(card, -1*zIndex);
-
+                Canvas.SetZIndex(card, 4);
```

Comment:
This won't work on Uno platforms I don't believe we support ZIndex on panels other than Canvas, right @jeromelaban ?

Reply:
@kazo0 some platforms support it, like Skia or Android.  Others may not.